library/src/main/java/com/bumptech/glide/load/data/mediastore/ThumbnailStreamOpener.java

```diff
@@ -66,37 +68,43 @@ public int getOrientation(Uri uri) {
   }
 
   public InputStream open(Uri uri) throws FileNotFoundException {
```

Comment:
extracted cursor handling to separate method and move up validation to look like guards